symfony-di/core.xml

```diff
@@ -43,7 +43,7 @@
         </service>
 
         <service id="sulu_document_manager.slugifier" class="Symfony\Cmf\Api\Slugifier\CallbackSlugifier" public="false">
-            <argument>Ferrandini\Urlizer::urlize</argument>
+            <argument>Behat\Transliterator\Transliterator::transliterate</argument>
```

Comment:
This works, but as for me a way better solution would be to implement something like IntlLocaleAwareSlugifier, wich would utilize `ext-intl` and it's `Transliterator::transliterate()`